### PR TITLE
[WIP]better postgres support

### PR DESCRIPTION
**Pull request by superpoussin22**
### Describe the Pull Request

```
Postgres support was broken because off change in settings.py
It should solve the issue 
```

### Checklist for PR

- [X] Run MobSF unit tests and lint `tox -e lint,test`.
- [X] Tested Working on  Docker
- [X] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)

### Additional Comments (if any)

```
postgress support mainly for docker using docker-compose
```

### CODE REVIEW

File: scripts/postgres_support.sh [hunk 1]
```diff
@@ -5,9 +5,9 @@ echo "Postgres support : ${POSTGRES}"
 if [ "$POSTGRES" == True ]; then
  pip3 install psycopg2-binary
  #Enable postgres support
- sed -i '/# Sqlite3 suport/,/# End Sqlite3 support/d' ../MobSF/settings.py && \
- sed -i "/# Postgres DB - Install psycopg2/,/'''/d" ../MobSF/settings.py && \
- sed -i "/# End Postgres support/,/'''/d" ../MobSF/settings.py && \
+ sed -i '/# Sqlite3 suport/,/# End Sqlite3 support/d' ../MobSF/settings.py && \  
+ sed -i ':a;N;$!ba;s/# Postgres DB - Install psycopg2\n"""/# Postgres DB - Install psycopg2/g' ../MobSF/settings.py && \
```
Comment (ajinabraham) [inline scripts/postgres_support.sh:9]: @superpoussin22 What does `:a;N;$!ba;s` do ?

Comment (superpoussin22) [reply scripts/postgres_support.sh:9]: without that it is  impossible to remove the """ in settings.py

**Comment by ajinabraham** [comment]
@superpoussin22 Feel free to merge this if it's working as expected.

**Comment by ajinabraham** [comment]
@superpoussin22 Is this related?
https://github.com/MobSF/Mobile-Security-Framework-MobSF/pull/1419

**Comment by superpoussin22** [comment]
closed inf favor of #1490

**Pull request closed without merging**